### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/advanced/benchmarks.md`

### PR DESCRIPTION
**Pull request by ceb10n**
Added portuguese translation for `docs/pt/docs/advanced/benchmarks.md`

Any feedback is appreciated

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 67682f7a19cc2e750a0b15652e076e1cd537626e at: https://50c16bc1.fastapitiangolo.pages.dev

File: docs/pt/docs/advanced/benchmarks.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Benchmarks
+
+Benchmarks independentes da TechEmpower mostra que aplicações **FastAPI** rodando com o Uvicorn como <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">um dos frameworks Python mais rápidos disponíveis</a>, ficando atrás apenas do Starlette e Uvicorn (utilizado internamente pelo FastAPI).
```
Comment (oandersonmagalhaes) [inline docs/pt/docs/advanced/benchmarks.md:3]:
```suggestion
Benchmarks independentes da TechEmpower mostram que aplicações **FastAPI** rodando com o Uvicorn como <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">um dos frameworks Python mais rápidos disponíveis</a>, ficando atrás apenas do Starlette e Uvicorn (utilizado internamente pelo FastAPI).
```

File: docs/pt/docs/advanced/benchmarks.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+Porém, ao verificar benchmarks e comparações você deve prestar atenção ao seguinte.
```
Comment (oandersonmagalhaes) [inline docs/pt/docs/advanced/benchmarks.md:5]:
```suggestion
Porém, ao verificar benchmarks e comparações você deve prestar atenção ao seguinte:
```

File: docs/pt/docs/advanced/benchmarks.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## Benchmarks e velocidade
+
+Quando você verifica os benchmarks, é comum ver diversas ferramentas de diferentes tipos comparados como se fossem equivalentes.
+
+Especificamente, para ver o Uvicorn, Starlette e FastAPI comparados entre si (entre diversas outras ferramentas).
+
+Quanto mais simples o problema resolvido pela ferramenta, melhor será a performance. E a maioria das análises não realiza testes das funcionalidades adicionais que são oferecidas pela ferramenta.
```
Comment (oandersonmagalhaes) [inline docs/pt/docs/advanced/benchmarks.md:13]:
```suggestion
Quanto mais simples o problema resolvido pela ferramenta, melhor será a performance. E a maioria das análises não testa funcionalidades adicionais que são oferecidas pela ferramenta.
```

File: docs/pt/docs/advanced/benchmarks.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+A hierarquia é:
+
+* **Uvicorn**: um servidor ASGI
+    * **Starlette**: (utiliza Uvicorn) um microframework web
+        * **FastAPI**: (utiliza Starlette) um microframework para APIs com diversas funcionalidades adicionais para a construção de APIs, com validação de dados, etc.
+
+* **Uvicorn**:
+    * Terá a melhor performance, pois não possui muito código além do próprio servidor.
+    * Você não escreveria uma aplicação utilizando o Uvicorn diretamente. Isso significaria que o seu código teria que incluir pelo menos todo o código fornecido pelo Starlette (ou o **FastAPI**). E caso você fizesse isso, a sua aplicação final teria a mesma sobrecarga que teria caso estivesse utilizado um framework e minimizando o código e os bugs.
```
Comment (oandersonmagalhaes) [inline docs/pt/docs/advanced/benchmarks.md:23]:
```suggestion
    * Você não escreveria uma aplicação utilizando o Uvicorn diretamente. Isso significaria que o seu código teria que incluir pelo menos todo o código fornecido pelo Starlette (ou o **FastAPI**). E caso você fizesse isso, a sua aplicação final teria a mesma sobrecarga que teria se utilizasse um framework, minimizando o código e os bugs.
```

File: docs/pt/docs/advanced/benchmarks.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    * O FastAPI provê mais funcionalidades em cima do Starlette. Funcionalidades que você quase sempre precisará quando estiver construindo APIs, como validação de dados e serialização. E ao utilizá-lo, você obtém documentação automática sem custo nenhum (a documentação automática sequer adiciona sobrecarga nas aplicações rodando, pois ela é gerada na inicialização).
+    * Caso você não utilize o FastAPI e faz uso do Starlette diretamente (ou outra ferramenta, como o Sanic, Flask, Responder, etc) você mesmo teria que implementar toda a validação de dados e serialização. Então, a sua aplicação final ainda teria a mesma sobrecarga caso estivesse usando o FastAPI. E em muitos casos, validação de dados e serialização é a maior parte do código escrito em aplicações.
+    * Então, ao utilizar o FastAPI você está economizando tempo de programação, bugs, linhas de código, e você provavelmente terá a mesma performance (ou até melhor) que você teria caso você não o utilizasse (já que você teria que implementar tudo no seu código).
```
Comment (oandersonmagalhaes) [inline docs/pt/docs/advanced/benchmarks.md:33]:
```suggestion
    * Então, ao utilizar o FastAPI, você está economizando tempo de programação, evitando bugs, linhas de código, e provavelmente terá a mesma performance (ou até melhor) do que teria caso você não o utilizasse (já que você teria que implementar tudo no seu código).
```

**Review by oandersonmagalhaes — CHANGES_REQUESTED** [review]
Thanks, Rafael. Great translation! I've added some suggestions. 🚀

**Comment by ceb10n** [comment]
> Thanks, Rafael. Great translation! I've added some suggestions. 🚀

Thanks [oandersonmagalhaes](https://github.com/oandersonmagalhaes) ;) 
I accepted all suggestions ;)

**Comment by tiangolo** [comment]
📝 Docs preview for commit 1803c7619f604c9304ef86353af610c4cd564a52 at: https://7589cd6d.fastapitiangolo.pages.dev

**Review by oandersonmagalhaes — APPROVED** [review]
Thank you! 🚀

**Comment by alejsdev** [comment]
Nice!  Thanks for the help @ceb10n @oandersonmagalhaes  :partying_face:

**Pull request merged**